packages/ext-angular-modern-kitchensink/src/examples/Ripple/Ripple.html

```diff
@@ -0,0 +1,22 @@
+<container #item
```

Comment:
Does not seem to be same as react counterpart.
React just has <Container layout="vbox">
where as you have used hbox. Why is that?